[2021-09-27 15:58:57,012] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: day1_challenge2.print_log_message 2021-09-27T15:58:43.060442+00:00 [queued]>
[2021-09-27 15:58:57,050] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: day1_challenge2.print_log_message 2021-09-27T15:58:43.060442+00:00 [queued]>
[2021-09-27 15:58:57,051] {taskinstance.py:1095} INFO - 
--------------------------------------------------------------------------------
[2021-09-27 15:58:57,052] {taskinstance.py:1096} INFO - Starting attempt 1 of 1
[2021-09-27 15:58:57,055] {taskinstance.py:1097} INFO - 
--------------------------------------------------------------------------------
[2021-09-27 15:58:57,109] {taskinstance.py:1115} INFO - Executing <Task(PythonOperator): print_log_message> on 2021-09-27T15:58:43.060442+00:00
[2021-09-27 15:58:57,125] {standard_task_runner.py:52} INFO - Started process 38559 to run task
[2021-09-27 15:58:57,144] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'day1_challenge2', 'print_log_message', '2021-09-27T15:58:43.060442+00:00', '--job-id', '113', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/day1_challenge2.py', '--cfg-path', '/tmp/tmplupyruj8', '--error-file', '/tmp/tmparb9opeo']
[2021-09-27 15:58:57,169] {standard_task_runner.py:77} INFO - Job 113: Subtask print_log_message
[2021-09-27 15:58:57,378] {logging_mixin.py:109} INFO - Running <TaskInstance: day1_challenge2.print_log_message 2021-09-27T15:58:43.060442+00:00 [running]> on host 196580edc338
[2021-09-27 15:58:57,604] {taskinstance.py:1254} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Attasil Chitwirote
AIRFLOW_CTX_DAG_ID=day1_challenge2
AIRFLOW_CTX_TASK_ID=print_log_message
AIRFLOW_CTX_EXECUTION_DATE=2021-09-27T15:58:43.060442+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-09-27T15:58:43.060442+00:00
[2021-09-27 15:58:57,615] {day1_challenge2.py:13} INFO - Print log message call logging info.
[2021-09-27 15:58:57,617] {python.py:151} INFO - Done. Returned value was: None
[2021-09-27 15:58:57,667] {taskinstance.py:1219} INFO - Marking task as SUCCESS. dag_id=day1_challenge2, task_id=print_log_message, execution_date=20210927T155843, start_date=20210927T155857, end_date=20210927T155857
[2021-09-27 15:58:57,865] {local_task_job.py:151} INFO - Task exited with return code 0
[2021-09-27 15:58:57,945] {local_task_job.py:261} INFO - 1 downstream tasks scheduled from follow-on schedule check
